src/GitHub.Exports.Reactive/Collections/TrackingCollection.cs

```diff
@@ -540,7 +540,7 @@ ActionData ProcessItem(ActionData data, List<T> list)
                 if (newer != null)
                 {
                     // the object is "older" than the one we have, ignore it
-                    if (newer(item, old) > 0)
```

Comment:
Should probably update the comment as well, since it no longer matches the logic 😄